goss-testing/tests/ncn/goss-check-customer-mgm-address.yaml

```diff
@@ -0,0 +1,39 @@
+#
```

Comment:
This test already appears to be covered in the `check_bgp_neighbors_established.sh` script, which is called by the `goss-switch-bgp-neighbor-aruba-or-mellanox.yaml` test. Is there a reason to create a new test to test just that piece here?